app/perfaddresscache.h

```diff
@@ -42,15 +42,17 @@ class PerfAddressCache
 
     struct SymbolCacheEntry
     {
-        SymbolCacheEntry(quint64 offset = 0, quint64 size = 0, const QByteArray &symname = {})
+        SymbolCacheEntry(quint64 offset = 0, quint64 value = 0, quint64 size = 0, const QByteArray &symname = {})
             : offset(offset)
+            , value(value)
             , size(size)
             , symname(symname)
         {}
 
         bool isValid() const { return !symname.isEmpty(); }
 
         quint64 offset;
+        quint64 value;
```

Comment:
please add a comment:

/// st_value in elf symbol table entry

Reply:
Can you also please document what the difference to offset is? Which value should be used when? Isn't offset + size already enough to define the symbol and to get it disassembled? Why is the value needed too?